CODE_OF_CONDUCT.md

```diff
@@ -1,78 +1,79 @@
 # CICE Consortium: Contributor Covenant Code of Conduct
+_The Contributor Code of Conduct is for participants in our software projects and community._
 
 ## Our Pledge
-
-In the interest of fostering an open and welcoming environment, we as contributors and
-maintainers pledge to making participation in our project and our community a harassment-
-free experience for everyone, regardless of age, body size, disability, ethnicity, gender
-identity and expression, level of experience, nationality, political affiliation, veteran
-status, pregnancy, genetic information, personal appearance, choice of text editor or
-operating system, race, religion, or sexual identity and orientation, or any other
-characteristic protected under applicable US federal or state law. 
+In the interest of fostering an open and welcoming environment,
+we, as contributors, creators, stewards, and maintainers (participants), of the CICE Consortium
+pledge to make participation in our software, system or hardware project and community a safe, productive, 
+welcoming and inclusive experience for everyone.
+All participants are required to abide by this Code of Conduct.
+This includes respectful treatment of everyone regardless of age, body size, disability, ethnicity, gender identity or expression,
+level of experience, nationality, political affiliation, veteran status, pregnancy, genetic information, physical appearance, 
+race, religion, or sexual orientation, as well as any other characteristic protected under applicable US federal or state law.
 
 ## Our Standards
-
-Examples of behavior that contributes to creating a positive environment include:
-
-* Using welcoming and inclusive language
-* Being respectful of differing viewpoints and experiences
-* Gracefully accepting constructive criticism
-* Focusing on what is best for the community
-* Showing empathy towards other community members
-
-Examples of unacceptable behavior by participants include:
-
-* The use of sexualized language or imagery and unwelcome sexual attention or advances
-* Trolling, insulting/derogatory comments, and personal or political attacks
-* Public or private harassment
+Examples of behaviors that contribute to a positive environment include:
+
+* All participants are treated with respect and consideration, valuing a diversity of views and opinions
+* Be considerate, respectful, and collaborative
+* Communicate openly with respect for others, critiquing ideas rather than individuals and gracefully accepting criticism
+* Acknowledging the contributions of others
+* Avoid personal attacks directed toward other participants
+* Be mindful of your surroundings and of your fellow participants 
```

Comment:
I'm not sure what this one is for

Reply:
Probably from UCAR's for field campaigns and such. I've removed it. It could apply to Workshops or Tutorials in the future, but I think we're okay to remove it.